packs/feats/force-of-nature.json

```diff
@@ -28,7 +28,13 @@
             "remaster": true,
             "title": "Pathfinder Howl of the Wild"
         },
-        "rules": [],
+        "rules": [
```

Comment:
```
{
  "key": "FastHealing",
  "value": 5,
  "deactivatedBy": [
    "silver"
  ],
  "type": "regeneration"
}
```

Reply:
That was my first thought too. But RAW Regeneration also means, that the creature's dying condition never increases beyond 3 as long as Regeneration is active. So, using Regeneration instead of Fast Healing could probably mess up with some mods for combat automation.

Reply:
Fair,

I would add a toggle to handle the deactivation in the event the PC gets targeted by a Silver weapon.

Reply:
Good idea. Added.